/aruco_node:
  ros__parameters:
    marker_size: 0.1
    aruco_dictionary_id: DICT_4X4_1000
    image_topic: /camera/camera/color/image_raw
    camera_info_topic: /camera/camera/color/camera_info
    camera_frame: camera_color_optical_frame
    publish_tf: true
    marker_ids: [0]